khmer/_khmer.cc

```diff
@@ -4329,6 +4332,63 @@ labelhash_n_labels(khmer_KGraphLabels_Object * me, PyObject * args)
 
 static
 PyObject *
+labelhash_label_across_high_degree_nodes(khmer_KGraphLabels_Object * me,
+        PyObject * args)
+{
+    LabelHash * labelhash = me->labelhash;
+
+    const char * long_str;
+    khmer_HashSet_Object * hdn_o = NULL;
+    Label label;
```

Comment:
A Label is just an annotation for a node. For now it's just a uint64 for simplicity, though it could be defined as a base class and subclassed for various types in the future.